src/azure-cli-core/azure/cli/core/__init__.py

```diff
@@ -147,7 +147,8 @@ def save_local_context(self, parsed_args, argument_definitions, specified_argume
             args_str = []
             for name, value in local_context_args:
                 args_str.append('{}: {}'.format(name, value))
-            logger.warning('Command argument values saved to local context: %s', ', '.join(args_str))
+            logger.warning('Your preference of: %s now saved to local context. To learn more, type in az '
+                           'local-context --help', ', '.join(args_str) + ' is' if len(args_str) == 1 else ' are')
```

Comment:
The colon seems weird to me. How about

```suggestion
            logger.warning('Your preference of '%s' are now saved to local context. To learn more, type in az '
                           'local-context --help', ', '.join(args_str) + ' is' if len(args_str) == 1 else ' are')
```

Reply:
We usually use backticks (`` ` ``) to denote commands. See https://github.com/Azure/azure-cli/pull/13060.